.lycheeignore

```diff
@@ -0,0 +1,5 @@
+https://localhost
+http://localhost
+https://0.0.0.0
+http://0.0.0.0
+https://github.com/issues
```

Comment:
Why ignore GH issues?

Reply:
Scanner gets detected by github as bot, and gets 404. Bots should be using GH API.